client/asset/dcr/spv.go

```diff
@@ -1047,7 +1047,7 @@ func (w *spvWallet) rescan(ctx context.Context, fromHeight int32, c chan wallet.
 
 // PurchaseTickets purchases n tickets, tells the provided vspd to monitor the
 // ticket, and pays the vsp fee.
-func (w *spvWallet) PurchaseTickets(ctx context.Context, n int, vspHost, vspPubKey string, mixCfg *mixingConfig) ([]*asset.Ticket, error) {
+func (w *spvWallet) PurchaseTickets(ctx context.Context, n int, vspHost, vspPubKey string, isMixingEnabled bool) ([]*asset.Ticket, error) {
```

Comment:
This `isMixingEnabled` parameter ought to be the same one managed by the Native wallet instance. At least that is my understanding.